apps/studio/data/telemetry/send-event-mutation.ts

```diff
@@ -48,22 +38,21 @@ export const useSendEventMutation = ({
   'mutationFn'
 > = {}) => {
   const router = useRouter()
-  const usePostHogParameters = useFlag('enablePosthogChanges')
 
   const consent =
```

Comment:
nit - just wondering whether this can be shifted to sendEvent now, then we don't need to pass consent as a prop

previously i did this just because of the feature flag check

Reply:
but would apply for all the other telemetry mutations

Reply:
Moved!